roles/inetd/defaults/main.yml

```diff
@@ -2,4 +2,4 @@
 ---
 services: []
 state: enabled
-backup: no
+backup: false
```

Comment:
Why this was changed to false?

Reply:
there is no change in behavior here, it is just that the yaml linting followed by ansible for boolean like values, they prefer that is should be either `false` and `true` in this particular format/letter case.